### PR TITLE
Adds onTouchMove support

### PR DESCRIPTION
**Pull request by bkeating**
Fairly crude but Works for Me(tm)
react-before-after-slider is a wonderful component. I'm surprised that touch support was not already included.

As an end-user, there is nothing required to take advantage of this detection.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -107,15 +110,25 @@ export default class BeforeAfterSlider extends Component {
...
   }
 
-  _onMouseMoveWrapper = (event) => {
-    const { offsetX } = event.nativeEvent
+  _onMoveWrapper = (event) => {
+    event.preventDefault()
+    let { offsetX } = event.nativeEvent
+    if (!offsetX) {
+      const rect = event.target.getBoundingClientRect()
+      offsetX = event.targetTouches[0].pageX - rect.left
```
Comment (transitive-bullshit) [inline src/index.js]: Maybe you want `event.touches[0].clientX`?

**Comment by transitive-bullshit** [comment]
This is an awesome addition -- thanks, @bkeating!

**Comment by transitive-bullshit** [comment]
Also, I'm originally from Elkhorn, WI :)

**Comment by bkeating** [comment]
What?! That’s incredible. We’re grabbing beers at some point... or Mead.

**Comment by transitive-bullshit** [comment]
Published as `v1.0.3`.

I'm living in Brooklyn these days, otherwise I'd definitely be down.

Thanks!

**Pull request merged**